dev-haskell/svg-builder/svg-builder-0.1.1-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="DSL for building SVG"
+HOMEPAGE="https://github.com/diagrams/svg-builder.git"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=dev-haskell/blaze-builder-0.4:=[profile?] <dev-haskell/blaze-builder-0.5:=[profile?]
+	>=dev-haskell/hashable-1.1:=[profile?] <dev-haskell/hashable-1.4:=[profile?]
+	>=dev-haskell/semigroups-0.16.1:=[profile?] <dev-haskell/semigroups-0.20:=[profile?]
+	>=dev-haskell/text-0.11:=[profile?] <dev-haskell/text-1.3:=[profile?]
+	>=dev-haskell/unordered-containers-0.2:=[profile?] <dev-haskell/unordered-containers-0.3:=[profile?]
+	>=dev-lang/ghc-7.6.2:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-1.16.0
+"
```

Comment:
Your comment says you used hackage's revision. But this change does not change `.cabal`'s file. I would expect it to hold outdated version bounds.

Reply:
The change is seen as a revision in hackage, the package was bumped to "r2" (we missed "r1").

Thus, hackport (in all it's infinite awesomeness) recognizes this and generates the new ebuild, despite the version number being the same.

Reply:
Package revisions on hackage are done in a way that change `.cabal` filed in index file stored at `01-index.tar`. They don't update `.tar.gz` file of a released package. Currently hackport does not download revised `.cabal` file separately. It should.

Reply:
HAH, ok I finally get it.

```sh
ebuild ${haskell_repo}/dev-haskell/svg-builder-0.1.1-r1.ebuild prepare
cat /var/tmp/portage/dev-haskell/svg-builder-0.1.1-r1/work/svg-builder-0.1.1-r1/svg-builder.cabal | grep `base`
```

This shows what you're saying, that despite the updates on hackages *index*, the actual tarball hasn't been changed. Thus we still need the `cabal_chdeps` stuff.

I think I knew this, but for some reason I wasn't quite comprehending what you were saying. I can fix that in this commit and others, no problem.